content/post/abortion-bill.en.md

```diff
@@ -0,0 +1,83 @@
+
+Reading privacy policies of those apps is tedious, they can change frequently, and ultimately they can be deemed illegal by state authorities.
+
+Finally a good solution may only be to use an app that processes and stores this sensible data locally (in the smartphone) like Apple Health. But again, a change in the app code could make this strategy evolve negatively regarding privacy. Any full open source app could be the only less-risky solution. Except Euki and Mozilla Foundation-backed Drip app, all others are storing user health data in the cloud. Some apps only share anonymized data (stripped of identifying user information) with third parties; but some techniques like machine learning can allow re-identification of these data. Most apps may share information in order to “comply with or respond to law enforcement”. The extraterritoriality of U.S. laws could lead, then, some non-U.S. apps to be obliged to share their data with state police. Finally, and this is obvious, a good data is a data that is not shared (or collected !).
+
+A low-tech solution using a thermometer, a calendar, and a pencil remains the safest even if it is less attractive.
+
+The FTC sued Kochava on Monday, August 29, 2022 for selling the geolocation information of "hundreds of thousands" of cell phones to anti-abortion groups.  These groups have used geolocation data to identify women who have visited abortion clinics[^10]. The worst future is already now!
+
+#### Shop and e-shops
+
+Some users flag that shops/e-shops that store your buying history may also be a risk. Your purchase history of menstrual hygiene products can also be requested by state police to reveal an illegal abortion. Some propose to desynchronize your purchases of menstrual hygiene products, pay in cash, and cancel your loyalty card (because it is identifying the buyer).
+
+#### Recommendations for users
+
+This abortion law does not only impact the freedom of women, it may also impact their health[^11] [^12]. So it is important to give some advice to enable users to protect themselves.
+
+As usual we can advise to use search engines that consume less data, such as DuckDuckGo and Startpage, use encrypted messaging services such as Signal or ProtonMail, and virtual private networks (VPN) to hide your real location, not browse while identified on some services like Google, clear cookies at browser exit, configure your browser to request web sites not to track you... Use open source app solutions, or cloudless solutions is also a good classic advice. Avoid using social network to discuss or disclose any sensible topics that you may not want to be turned against you later on. Remember that the internet does not forget anything, nothing is really private and something that you said 10 years ago could be an issue in a far future.
```

Comment:
```suggestion
As usual we can advise to use search engines that consume less data, such as DuckDuckGo and Startpage, use encrypted messaging services such as Signal or ProtonMail, and virtual private networks (VPN) to hide your real location, not browse while identified on some services like Google, clear cookies at browser exit, configure your browser to request web sites not to track you… Use open source app solutions, or cloudless solutions is also a good classic advice. Avoid using social network to discuss or disclose any sensible topics that you may not want to be turned against you later on. Remember that the Internet does not forget anything, nothing is really private and something that you said 10 years ago could be an issue in a far future.
```
Consume data seems strange, maybe collect would be better?
Don't browse seems better.
DNT was a good idea but most websites declares they just don't care and don't pay attention to it.
Why use "social network" here when "social media" was used above?
Finally, as noted in the French part , telling people to use VPN if they don't understand what is it and the consequences could do more harm than good if they dangerously keeps too much data for example.

Reply:
Changed "collect". Added "trustworthy VPN". Changed "social network" to media too. For the "not/don't browse" the global sentence is "we can advise to not browse" so no auxiliary here.